auth-web/src/views/auth/RequestNameView.vue

```diff
@@ -16,36 +16,41 @@
           <v-icon size="8" class="list-item-bullet mt-5">mdi-square</v-icon>
           <v-list-item-content>
             <v-list-item-subtitle class="list-item-text">
-              You can choose to incorporate a numbered Benefit Company
+              You can choose to incorporate a <numbered-company-tooltip />
               and start your incorporation immediately.
             </v-list-item-subtitle>
           </v-list-item-content>
         </v-list-item>
         <!-- Panel Btns -->
         <div class="request-name-info-btns mt-5">
           <v-btn large color="#003366" class="white--text"
-            href="https://www.bcregistrynames.gov.bc.ca/nro/" target="_blank" rel="noopener noreferrer">
+                 :href="nroUrl" target="_blank" rel="noopener noreferrer">
             Request a Name
           </v-btn>
           <p class="mt-5">Have an existing Name Request?
-            <a href="https://www.bcregistrynames.gov.bc.ca/nro/nro.htm?_flowId=anonymous-monitor-flow&_flowExecutionKey=e1s1"
+            <a :href="`${nroUrl}nro.htm?_flowId=anonymous-monitor-flow&_flowExecutionKey=e1s1`"
               target="_blank" rel="noopener noreferrer" class="status-link">
               Check your Name Request Status
             </a>
           </p>
-          <LearnMoreButton />
+          <learn-more-button
+            :redirect-url="learnMoreUrl"
+          />
         </div>
       </v-col>
       <!-- Image Column -->
       <v-col cols="12" md="6">
-        <v-img src="../../assets/img/Step2-Name-Request.png" aspect-ratio="1.2" contain></v-img>
+        <a :href="nroUrl" target="_blank">
```

Comment:
You know what would be really sexy (if we had time)? A new component type or directive to suppose the DRY principle (since it seems there are lots of anchor links to NR). Just an idea.

Reply:
Is a great idea, i was thinking the same after i did this a few times, i realized my folly. Will update this now!

Reply:
It's an academic exercise. Only do if there is time :)

Reply:
Created a Name Request Button component and implemented where it's used 👍 
Tho, there was still one instance where i pulled the nroUrl other than the component.